src/pages/en/reference/api-reference.md

```diff
@@ -506,7 +510,9 @@ import { Prism } from '@astrojs/prism';
 <Prism lang="js" code={`const foo = 'bar';`} />
 ```
 
-> **`@astrojs/prism`** is built-in as part of the `astro` package. No need to install as a separate dependency just yet! However, note that we do plan to extract `@astrojs/prism` to a separate, installable package in the future.
+:::note
+**`@astrojs/prism`** is built-in as part of the `astro` package. No need to install as a separate dependency just yet! However, note that we do plan to extract `@astrojs/prism` to a separate, installable package in the future.
```

Comment:
:::caution[Deprecated]
**`@astrojs/prism`** will be extracted to a separate, installable package in the future.